src/components/predictive-layer/full-width-predictive.vue

```diff
@@ -0,0 +1,225 @@
+<template>
```

Comment:
Should the remove buttons of the `history-queries` be aligned to the right?

![image](https://github.com/empathyco/x-archetype/assets/75546736/c7c425dd-35e8-436e-aa83-a875c412f7c9)